### PR TITLE
[ut] add ut for binding.go

### PR DESCRIPTION
**Pull request by ikaven1024**
Signed-off-by: yingjinhui <yingjinhui@didiglobal.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
add ut for  pkg/util/helper/binding.go

**Which issue(s) this PR fixes**:
Part of #2579

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/util/helper/binding_test.go [hunk 1]
```diff
@@ -180,3 +196,695 @@ func TestObtainBindingSpecExistingClusters(t *testing.T) {
...
+							},
+						},
+					},
+				).Build(),
+				namespace: "default",
+				name:      "foo",
+			},
+			want:    []workv1alpha1.Work{},
+			wantErr: false,
+		},
```
Comment (XiShanYongYe-Chang) [inline pkg/util/helper/binding_test.go:749]: Can here add more cases: the deleted object not found, delete failed?

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~

File: pkg/util/helper/binding_test.go (hunk 1 above)
Comment (ikaven1024) [reply pkg/util/helper/binding_test.go:760]: I don't find a way to simulate delete fail with fake client~

Comment (XiShanYongYe-Chang) [reply pkg/util/helper/binding_test.go:760]: Okay, I see.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**